docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
+services:
+  gateway:
+    image: 267292272963.dkr.ecr.us-east-1.amazonaws.com/uptycsdev/presto-gateway/presto-gateway:v1.3
+    container_name: prestogateway
+    ports:
+      - 8082:8082
+    volumes:
+      - ${PRESTOGATEWAY_HOME}/config:/opt/uptycs/prestogateway/config
```

Comment:
Can we move this to /opt/uptycs/etc/prestogateway/config
Configs for all other services are under etc only.
log folder can also be mounted.. the path is /opt/uptycs/var/log/

Reply:
this docker compose file is for local setup only.  for actual deployment on stack it is part of edna . I will change  volume path there for config and logs